Online-Messaging-Backend/src/routes/profiles/ProfileDAO.ts

```diff
@@ -0,0 +1,105 @@
+/* tslint:disable:no-console */
+import aws from "aws-sdk";
+import {awsConfigPath} from "../../config/aws-config";
+
+aws.config.loadFromPath(awsConfigPath);
+
+const docClient = new aws.DynamoDB.DocumentClient();
+
+const table = "Profiles";
+
+class ProfileDAO {
+    public createProfileFromUser(username: string, email: string, firstName: string, lastName: string): Promise<any> {
```

Comment:
The Users table already has all this information, we don't need to re-submit to a new Profiles table